releasenotes-dca/notes/add_generic_k8s_resource_tagging-64b51a2ffaf3e2cc.yaml

```diff
@@ -0,0 +1,16 @@
+# Each section from every release note are combined when the
+# CHANGELOG-DCA.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Added capability to tag any kubernetes resource based on labels and annotations.
+    This feature can be configured via `kubernetes_resources_annotations_as_tags` and `kubernetes_resources_labels_as_tags`.
+    These feature configurations are maps from group resource to annotations/labels to tag map.
+    For example, `deployments.apps` can be mapped to an annotations to tags map to configure annotations as tags for deployments.
```

Comment:
```suggestion
    These feature configurations are maps from group resource to a map of annotations/labels to tag.
    For example, `deployments.apps` can be mapped to an annotations-to-tags map to configure annotations as tags for deployments.
```

Reply:
This suggestion is based on the assumption that `annotations to tags` is an adjective describing the map, but please let me know if that's not the case

Reply:
Indeed, your understanding is correct. 

The same applies to this part `annotations/labels to tag map.`, but I am not certain if `annotations/labels-to-tag map` is acceptable.